packages/expo-localization/src/Localization.ts

```diff
@@ -62,7 +68,7 @@ export const isRTL = ExpoLocalization.isRTL;
 
 // @needsAudit
 /**
- * Consider using Localization.getLocales() for a list of user preferred locales instead.
+ * @deprecated Use Localization.getLocales() instead.
```

Comment:
```suggestion
 * @deprecated Use `Localization.getLocales()` instead.
```

Reply:
We should also link the recommended replacement in the notice, if possible.

Reply:
Yes, great point. I just updated the suggestion.